### PR TITLE
ceph_test_rados: make long name ~300 chars, (not ~800)

### PR DESCRIPTION
**Pull request by liewegas**
Signed-off-by: Sage Weil <sage@redhat.com>

### CODE REVIEW

File: src/test/osd/TestRados.cc [hunk 1]
```diff
@@ -56,10 +56,7 @@ class WeightedTestGenerator : public TestOpGenerator
       oid << m_op;
       if (m_op % 2) {
 	// make it a long name
-	oid << " ";
-	for (unsigned i = 0; i < 300; ++i) {
-	  oid << i;
-	}
+	oid << " " << string(300, 'o');
```
Comment (tchaikov) [inline src/test/osd/TestRados.cc]: @liewegas i don't get it, we append 301 spaces to oid. and we append a string of 301 spaces. so nothing changed in this patch. it's but a refactor unlike claimed by the commit message?

Comment (liewegas) [inline src/test/osd/TestRados.cc]: we were appending "1" "2" "3" "4" ... "299" "300".

Comment (tchaikov) [inline src/test/osd/TestRados.cc]: oh, right!

**Comment by tchaikov** [comment]
lgtm.

**Pull request merged**